src/test/kotlin/org/rust/ide/annotator/RsAnnotationTestBase.kt

```diff
@@ -120,4 +124,19 @@ abstract class RsAnnotationTestBase : RsTestBase() {
         myFixture.configureFromTempProjectFile(testFilePath)
         myFixture.testHighlighting(false, checkInfo, false)
     }
+
+    protected fun checkByCodeFragment(
+        @Language("Rust") context: String,
+        fragment: String,
+        fragmentConstructor: (Project, String, RsElement) -> RsCodeFragment,
```

Comment:
Maybe reuse `FragmentConstructor` since we introduced it?

Reply:
I decided to get rid of this typealias, looks like it is not necessary and doesn't really improve the readability